markdown.js

```diff
@@ -7,17 +7,24 @@
  * input files that require this compiler
  */
 
-module.exports = {
-  applies_to: applies_to,
-  compile: compile
-};
+module.exports = setup;
 
 var md = require("marked");
 
-function compile (src) {
-  return md(src);
-}
+function setup (config) {
+	var settings = config.marked_settings || {};
+	md.setOptions(settings);
+
+	return {
+	  applies_to: applies_to,
+	  compile: compile
+	};
+
+	function compile (src) {
+	  return md(src);
+	}
```

Comment:
It appears as if your editor has set indentation to 8 spaces. All files in this project use 2 spaces.